src/device/libpixy_m4/src/misc.cpp

```diff
@@ -106,24 +106,38 @@ void showError(uint8_t num, uint32_t color, const char *message)
         if (message)
             lpc_printf(message);
     }
- }
+}
 
-uint16_t crc16(const void* const data, const uint32_t size, uint16_t init)
+uint8_t crc8(const void* const data, uint8_t len)
 {
-    uint16_t crc = init;
-    const uint8_t* const buffer = (const uint8_t* const)data;
-
-    for (unsigned i = 0; i < size; i++)
+    static uint8_t table[256] = {
```

Comment:
Could you please add a note explaining the purpose of this table

Reply:
done